routes/clucks.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const multer = require('multer');
+const express = require('express');
+const router = express.Router();
+const knex = require('../db');
+
+const UPLOADS_DIR = 'uploads';
+const upload = multer({dest: path.join(__dirname, '..', 'public', UPLOADS_DIR)});
+
+// PATH: /clucks/new VERB: GET Serves form for creating clucks
+router.get('/new', (request, response) => {
+  response.render('clucks/new');
+});
+
+// localhost:7000/clucks/:id
+router.get('/:id', (request, response) => {
+  const id = request.params.id;
+  knex
+    .first()
+    .from('clucks')
+    .where({id})  //{id} === {id: id}
+    .then(cluck => {
+      response.render('clucks/show',{cluck});
```

Comment:
You never made a 'clucks/show' view, so your code is breaking when you try to access the show page for a cluck. I would have not included links to the show view from my index view since you did not implement a show view